packages/compass-app-stores/src/stores/instance-store.js

```diff
@@ -265,6 +265,10 @@ store.onActivated = (appRegistry) => {
     appRegistry.emit('select-namespace', metadata);
   });
 
+  appRegistry.on('database-dropped', async() => {
```

Comment:
Does this event also fire when another database/collection is open and the dropped database was just a random one from the sidebar?

Reply:
So yeah you could currently be viewing a different database. Don't know if/how we could detect that.